# empty request, as the node listens for incoming point clouds on its own
int32 EMPTY = 1
int32 FULL = 2
int32 call

---

# The graph of the clusters 
static_segmentation/StaticSeg[] graph_queue

# The graph of the clusters 
sensor_msgs/Image graph_image

# The collection of masks
sensor_msgs/Image[] cluster_masks

# Whether the detection has succeeded or failed
int32 FAILURE = 1
int32 SUCCESS = 2
int32 result
